### PR TITLE
Adds a few missing directives to the index, and fixes some discrepancies

### PR DESCRIPTION
**Pull request by darrellwarde**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-graphql-27.surge.sh

File: modules/ROOT/pages/type-definitions/directives/index.adoc [hunk 1]
```diff
@@ -11,6 +11,12 @@ The Neo4j GraphQL Library provides the following directives to be used whilst de
 | xref::/type-definitions/directives/database-mapping.adoc#type-definitions-alias[`@alias`]
 | Maps a GraphQL schema field to a Neo4j property on a node or relationship.
 
+| xref::/authentication-and-authorization/authentication.adoc[`@authentication`]
+| Require authentication checks when accessing the type.
```
Comment (lidiazuin) [inline modules/ROOT/pages/type-definitions/directives/index.adoc:15]:
```suggestion
| Requires authentication checks when accessing the type.
```

File: modules/ROOT/pages/type-definitions/directives/index.adoc [hunk 2]
```diff
@@ -11,6 +11,12 @@ The Neo4j GraphQL Library provides the following directives to be used whilst de
...
+
+| xref::/authentication-and-authorization/authorization.adoc[`@authorization`]
+| Specify authorization rules for queries and mutations on the type.
```
Comment (lidiazuin) [inline modules/ROOT/pages/type-definitions/directives/index.adoc:18]:
```suggestion
| Specifies authorization rules for queries and mutations on the type.
```

File: modules/ROOT/pages/type-definitions/directives/index.adoc [hunk 3]
```diff
@@ -68,6 +74,9 @@ Can only be used on interfaces, as per its definition:
 directive @relationshipProperties on INTERFACE
 ----
 
+| `@relayId`
+| Specify that the field should be used as the global node identifier for Relay.
```
Comment (lidiazuin) [inline modules/ROOT/pages/type-definitions/directives/index.adoc:78]:
```suggestion
| Specifies that the field should be used as the global node identifier for Relay.
```

File: modules/ROOT/pages/type-definitions/directives/index.adoc [hunk 4]
```diff
@@ -77,6 +86,9 @@ directive @relationshipProperties on INTERFACE
 | xref:/schema-configuration/type-configuration.adoc#_subscription[`@subscription`]
 | Limits subscription operations in the library.
 
+| `@subscriptionsAuthorization`
+| Specify authorization rules for subscriptions on the type.
```
Comment (lidiazuin) [inline modules/ROOT/pages/type-definitions/directives/index.adoc:90]:
```suggestion
| Specifies authorization rules for subscriptions on the type.
```

**Review by lidiazuin — APPROVED** [review]
Thanks!

**Pull request merged**